.github/workflows/publish_pypi.yml

```
@@ -0,0 +1,53 @@
+name: Publish
+
+on:
+  push:
+    tags: ["[0-9]+.[0-9]+.[0-9]+"]
```

```suggestion
    tags: ["[0-9]+\.[0-9]+\.[0-9]+"]
```
Regex `.` matches any character.